specification/mariadb/resource-manager/Microsoft.DBforMariaDB/preview/2018-06-01-preview/examples/RecoverableServersGet.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "resourceGroupName": "testrg",
+    "serverName": "testsvc4",
+    "api-version": "2018-06-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/testrg/providers/Microsoft.DBforMariaDB/servers/testsvc4/recoverableServers",
+        "name": "recoverableServers",
+        "type": "Microsoft.DBforMariaDB/servers/recoverableServers",
+        "properties": {
+          "edition": "GeneralPurpose",
+          "serviceLevelObjective": "GP_Gen5_2",
+          "vCore": 2,
+          "hardwareGeneration": "Gen5",
+          "lastAvailableBackupDateTime": "2020-11-20T01:06:29.78Z",
+          "version": "10.3"
+        }
```

Comment:
List API must return an array. Is this really what the API returns?

Reply:
Yes this is what the service returns

This changes are to align with the service for S360 violations.  The service has been in GA for 3 years and there are no changes at the service side and we are just adding the missing API back to the swagger.  These API were only required from the portal perspective and no customers consuming this.  We are not doing any future development in this platform and is in the sunset path.